### PR TITLE
446 fixed avro batch offset commit

### PR DESCRIPTION
**Pull request by wojtkiewicz**

### CODE REVIEW

**Comment by druminski** [comment]
👍  I also tested this fix by custom integration test with avro topic. During the test I saw correct kafka topic name in a place where offsets are commited.

File: hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/batch/MessageBatchReceiver.java [hunk 1]
```diff
@@ -106,7 +106,7 @@ private Message receive(Subscription subscription, String batchId) {
 
     private MessageMetadata messageMetadata(Subscription subscription, String batchId, Message next) {
```
Comment (tmmi) [inline hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/batch/MessageBatchReceiver.java]: why Message next ?

Comment (wojtkiewicz) [inline hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/batch/MessageBatchReceiver.java]: fixed

**Comment by pbetkier** [comment]
👍

**Pull request merged**